compiler/rustc_error_messages/locales/en-US/metadata.ftl

```diff
@@ -4,6 +4,10 @@ metadata_rlib_required =
 metadata_lib_required =
     crate `{$crate_name}` required to be available in {$kind} format, but was not found in this form
 
+metadata_rustc_lib_required =
+    crate `{$crate_name}` required to be available in {$kind} format, but was not found in this form
```

Comment:
It was unclear to me at first why this suggested `rustc_driver` unconditionally - the error message seems like a better place for it than the comment in `src/test/run-make-fulldeps`.
```suggestion
    crate `{$crate_name}` required to be available in {$kind} format, but was not found in this form
    .note = only .rmeta files are distributed for `rustc_private` crates other than `rustc_driver`
```